### ISSUE TITLE
PC build on debian build, but could not launch browser or something

### ISSUE THREAD

**Issue by yoyz** [mtorromeo/gmenu2x#7]
Hi mtorromeo,

First thank you for this great apps.
I decide to build it on my debian, to launch it and see how it is done to learn from it.
I like this little app, it's well done, and beautifull.

To reach the binary I had to modify : 
- font.cpp to give it good path, and it works, I have font ;
- add the -lpng to the link stage ;
- get Paul Cercueil libini ;
- comment line 256,257,258 in menu.cpp, related to a "maxClock", build issue ;

Then with the help of an old zip gmenu2x-gp2x-0.11.zip, I've put some file and directory in ~/.gmenu2x : skins and sections fetch from a gp2x binary. And I've constructed my input.cnf

BUT
I'm not able to change from setting to emulator for example or enter any menu.
I can create a "emulator" section with a mkdir command in ~/.gmenu2x.
I'm only able to "show the menu" which show me "add link in setting", I can move in the menu, go back, but I can not enter any thing. Space or enter, do nothing and they are catch by SDL, so... I don't understand what's wrong...

Am I missing an obvious thing ? 
I've got a open2x, gp2x, and it works fine, so I know it's my environment, but I don't see what. 
I compile with -DLOG_LEVEL=4

I use this input.conf

confirm=keyboard,32
cancel=keyboard,27
section_prev=keyboard,280
section_next=keyboard,281
#pageup=keyboard,279
#pagedown=keyboard,281
menu=keyboard,319
settings=keyboard,13
up=keyboard,273
down=keyboard,274
left=keyboard,276
right=keyboard,275
dec=keyboard,45
inc=keyboard,61

I don't understand what is broken, or if I miss a basic -DSOMESTUFF in the compilation for PC.
Hope you have an idea :)
Have a nice day, great software.

**Comment by mtorromeo** [comment]
HI,
I'm glad you like it.

Sadly I didn't have the time and motivation to continue GMenu2X development and as you can see the last commits are from 6 years ago.

In theory you shouldn't have to copy files from other zips to make it  work.
After compiling the software with `make` you can run `make dist` to build a complete directory layout under `dist/pc/gmenu2x` and run it from there.

If it still doesn't work, my first guess would be that the libraries you are linking against on your PC are more up to date than the ones in open2x and you are being affected by some incompatibilities that should be addressed.

**Comment by yoyz** [comment]
Hi mtorromeo,

I understand this program take you lots of time and after all years, it works, so there is nothing to add, and it is great.

I discover, that I've been a complete mess, I was sure I was on your repo, but, no, I was on this one https://github.com/Nebuleon/gmenu2x-gcw0.
I've been able to build it, the only thing I had to do is to modify the makefile to use /usr/lib64 instead of /usr/lib, and libpng instead of libpng12 ( or this kind of number).
make dist works fine.
I put everything in dist/pc/gmenu2x in ~/.gmenu2x.
I launch cd ~/.gmenu2x, then the binary from ~/.gmenu2x and it works fine.
So great program :)  Great packaging, :) And thank you for your help :)
